tools/codegen/gen_lazy_tensor.py

```diff
@@ -223,15 +228,20 @@ def gen_key(func: FunctionSchema) -> Tuple[str, str]:
     assert class_name is not None
 
     # Generate nativefunction declarations
+    # Note, eager registrations is set to False for the lazy TS backend as another LTC backend
+    # may want to register their own lazy kernels instead of registering the TS ones.
+    # The registration will lazily happen when init_ts_backend is called.
     gen_dispatchkey_nativefunc_headers(fm, class_name, cpp_namespace, backend_indices,
-                                       grouped_native_functions, backend_key, autograd_key)
+                                       grouped_native_functions, backend_key, autograd_key,
+                                       backend_name, eager_registration=False)
```

Comment:
@wconstab I think there's a problem here. According to the [function definition](https://github.com/pytorch/pytorch/blob/641cab9a7097d1769fccd4aef8300302ffa2fb83/tools/codegen/gen_backend_stubs.py#L204-L213) of `gen_dispatchkey_nativefunc_headers`, the positional argument after `autograd_dispatch_key` is `eager_registration`, but instead you passed in `backend_name`. Since you also set the value of `eager_registration` using a keyword, the following error occurs:

```
TypeError: gen_dispatchkey_nativefunc_headers() got multiple values for argument 'eager_registration'
```